packages/admin/admin/src/dataGrid/muiGridSortToGql.tsx

```diff
@@ -7,12 +7,12 @@ type SortEntry = {
     direction: "ASC" | "DESC";
 };
 
-export function muiGridSortToGql(sortModel?: GridSortModel, apiRef?: ReturnType<typeof useGridApiRef>) {
+export function muiGridSortToGql(sortModel?: GridSortModel, columns?: GridColDef[]) {
```

Comment:
I believe @jamesricky intentionally used the apiRef for this, but I can't remember why 🤔

Reply:
in my opinion, it's bad practice to add this heavy type into this kind of function. The function should take as few and as specific arguments as possible.

If there is a good reason, we can for sure revert this - but due to the more complex type that mui introduced, GridColDef - it's probably not easy and nice to fix.

Reply:
I can't think of a reason not to just use `columns`, so can't have been too important 😄

Maybe it had something to do with a feature that wasn't supported by the old version of the free `DataGrid`. Or maybe I just didn't do it intentionally 😅